rtgui/thumbimageupdater.cc

```diff
@@ -164,10 +164,10 @@ class ThumbImageUpdater::Impl :
         }
 
         if ( --active_ == 0 ) {
-            Glib::Threads::Mutex::Lock lock(mutex_);
+            std::lock_guard<std::mutex> lock(mutex_);
             if (inactive_waiting_) {
                 inactive_waiting_ = false;
-                inactive_.broadcast();
+                inactive_.notify_one();  // notify_all ?
```

Comment:
Don't know. Is a thundering herd allowed here?

Reply:
`notify_all` is the proper equivalent here.